docs/Tutorial/getting-started-md

```diff
@@ -0,0 +1,128 @@
+# Quick guide
+
+## What you'll do
+
+- Deploy the nginx-ingress-integrator charm
+- Relate it to another charm
+- Inspect the ingress it creates.
+
+We'll then also look at changing that ingress via a juju configuration update.
+
+## Requirements
+
+You will need:
+
+* A laptop or desktop running Ubuntu (or you can use a VM).
+* [Juju and Microk8s](https://juju.is/docs/olm/microk8s) installed. We’ll also want to make sure the ingress add-on is enabled, which we can do by running `microk8s enable ingress`.
+
+## Deploy this charm
+
+To deploy the charm and relate it to
+[the Hello Kubecon charm](https://charmhub.io/hello-kubecon) within a Juju Kubernetes model:
+
+    juju deploy nginx-ingress-integrator
+    juju deploy hello-kubecon
+    juju relate nginx-ingress-integrator hello-kubecon
+    # If your cluster has RBAC enabled you'll be prompted to run the following:
+    juju trust nginx-ingress-integrator --scope cluster
+
+Once the deployment has completed and the "hello-kubecon" workload state in
+`juju status` has changed to "active" you test the application in
+a browser. 
+
+To do so, we need to find the IP address of the ingress controller, which we can do by running `microk8s kubectl get pods -n ingress -o wide`. In my case the output was:
+```
+NAME                                      READY   STATUS    RESTARTS       AGE   IP             NODE        NOMINATED NODE   READINESS GATES
+nginx-ingress-microk8s-controller-c4vp9   1/1     Running   2 (119s ago)   17h   10.1.129.161   finistere   <none>           <none>
+```
+So now I would edit `/etc/hosts` on my laptop and add `10.1.129.161    hello-kubecon`. I could then open a new browser window and visit `http://hello-kubecon`.
```

Comment:
Do you think the wording could be improved here? Perhaps "Adding `10.1.129.161    hello-kubecon` to `/etc/hosts` lets you visit `http://hello-kubecon` and see the `hello-kubecon` charm in action!"